index.html

```diff
@@ -0,0 +1,262 @@
+                        </div>
+                        <h6 class="h33">A daily selection of privately
+                             personalized reads; no accounts 
+                             or sign-ups required.
+                        </h6>
+                        
+                            <ul class="p-languages2">
+                                <li class="language2">HTML</li>
+                                <li class="language2">CSS</li>
+                                <li class="language2">JavaScript</li>
+                             </ul>
+                             <button class="button2" type="button">See Project</button>
+                            
+                    </div>
+
+                </div> 
+           </div>
+        </section>
+        <section class="Works" >
+            <div  class="workflex">
+                <div class="workgrid">
+                    <div class="area1">
+                       <img src="./img/mutipost.png"> 
+                    </div>
+                    <div class="area2">    
+                        <h8 > Multi-p Stories</h8>
+                        <div class="workchildx">                           
+                                <ul  class="workchildx">
+                                    <li class="conopy">CANOPY</li>
+                                    <li class="counter">.</li>
+                                    <li class="backend">Back End Dev</li>
+                                    <img class="counter" src="./img/Counter.png" alt="counter" />
+                                    <li class="year2015">2015 </li>
+                                </ul>                    
+                        
+                        </div>
+                        <h6 class="h33">A daily selection of privately
+                             personalized reads; no accounts 
+                             or sign-ups required.
+                        </h6>
+                        
+                            <ul class="p-languages2">
+                                <li class="language2">HTML</li>
+                                <li class="language2">CSS</li>
+                                <li class="language2">JavaScript</li>
+                             </ul>
+                             <button class="button2" type="button">See Project</button>
+                            
+                    </div>
+
+                </div> 
+           </div>
+        </section>
+        <section class="footerbar  ">         
+
+            <section class="flex-bottom">            
+                <div class="footer">
+                <h2>About Myself<h2>
+                </div>
+                <div class="headerdetails">
+                    <h class="h9">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</h>
```

Comment:
- [ ] I have noticed you are making use of an incorrect heading tag. please kindly note that HTML standards only provides for 6 heading tags which are `h1, h2, h3, h4, h5, h6`. follow this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) to learn more about this :+1: